actionmailer/lib/action_mailer/test_case.rb

```diff
@@ -21,9 +21,12 @@ module Behavior
 
       included do
         class_attribute :_mailer_class
+        class_attribute :fixture_path, instance_writer: false
         setup :initialize_test_deliveries
         setup :set_expected_mail
         teardown :restore_test_deliveries
+
+        self.fixture_path = File.join(Rails.root, 'test', 'fixtures')
```

Comment:
Not sure `Rails.root` is guaranteed to be available when this module is included. I remember us changing a couple spots to get rid of `Rails.root` references to make the individual framework components independent. The option should probably be set in the Railtie. @rafaelfranca should know the details.

Reply:
I seconded this. We should set this value in action mailer's Railtie.